doc/reference/networking/dhcpv4.rst

```diff
@@ -1,11 +1,24 @@
 .. _dhcpv4_interface:
 
-DHCP v4
-#######
+DHCPv4
+######
 
 Overview
 ********
 
+The Dynamic Host Configuration Protocol (DHCP) is a network management protocol
+used on IPv4 networks. A DHCPv4 server dynamically assigns an IPv4 address
+and other network configuration parameters to each device on a network so they
+can communicate with other IP networks.
+See https://en.wikipedia.org/wiki/Dynamic_Host_Configuration_Protocol for
```

Comment:
This would look better in the rendered HTML:
```suggestion
See this `DHCP Wikipedia article <https://en.wikipedia.org/wiki/Dynamic_Host_Configuration_Protocol>`_ for
```